kctf-operator/pkg/controller/challenge/set/default.go

```diff
@@ -1,5 +1,5 @@
 // File that set values correctly and return default values that weren't specified
-package challenge
+package set
```

Comment:
same for functions in this package: without word Set is cleaner

Reply:
Same thing as numReplicas, it makes sense for the last function which is the one used in other package but the three first ones are of internal usage. Do you think it's better to leave without the set (like setPersistentVolumeClaim)?

Reply:
same here, you can leave it